app/graphql/mutations/user/add_favorite_movie.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+module Mutations
+  module User
+    class AddFavoriteMovie < AuthBase
+      type Types::UserAddFavoriteMovieType
+
+      description I18n.t('graphql.mutations.user.add_favorite_movie.desc')
+
+      argument :input, Types::Inputs::UserAddFavoriteMovieInput, required: true
+
+      def resolve(input:)
+        favorite_movie = FavoriteMovie.create(movie_id: input[:movie_id], user_account: current_user)
```

Comment:
I would use a trailblazer operation here: `match_operation ::User::Movie::Operation::AddFavorite.call(...)` to validate that  the movie with `movie_id` exists and it's not in the list of favorites already

Reply:
I understand what you're talking about and logic should be stored in services/trailblazer operations in commercial projects, but it's test task and I would keep logic here because it's about learning GraphQL, not Trailblazer.